compact/definitions.xml

```diff
@@ -303,7 +303,7 @@ The unused IDs below are saved for future use.
     <constant name="TaggerCalorimeter_ID"    value="199"/>
 
     <documentation>
-      #### (200-219) Far Backward Beamline Magnets
+      #### (200-227) Far Backward Beamline Magnets and Pipes
```

Comment:
Rather than adding 30 IDs for not really any new detector, can we try to figure out a different solution here for passive elements? Can we just stick this bunch of Pipes inside a single detector with one ID?

Reply:
Ok.  Now the magnets have been consolidated to use far fewer IDs.

Reply:
Wouter, does my solution of chaining together multiple magnets/beam pipes into single IDs work to address your concern?  The ones I added are 212-215.  The other IDs for pieces of beam pipes and magnets were there before (I think from Simon).  These could be chained together as well in another PR.

Reply:
Yes, it does. But I think this is now not strictly a far backward class. Would it make sense to rename it to something more generic to make clear that others can use it too? I'm imagining some far forward people @ajentsch might need to implement some beamlines that are currently taking up some valuable detector ID space.

Reply:
Changed the names accordingly.

Reply:
What is the motivation behind giving detectors without any sensitive elements IDs in the first place? The components in my previous hadron beamline extension haven't got any (this should of course be changed if necessary).

Reply:
I am not absolutely sure they are necessary for passive elements but there is some "ID" needed here:
https://github.com/eic/epic/blob/9526e811921e4447c5fa1cfbc9a1a198b3fc7efb/src/CylindricalMagnetChain_geo.cpp#L27

Reply:
So, is it not needed to put an ID in the constructor of a DetElement or can it stay the way I have it?

Reply:
I'm not certain, the way I've put the hadron beampipe in at the moment doesn't give an id in the xml but it's still used in the DetElement constructor so there must be a default. It does seems wasteful to require every geometry in the simulation to have separate and limited ids when as far as I can tell they only ever appear in readout. Does it appear in debug statements if tracks get stuck or the such?
Hoping @wdconinc or @veprbl will explain why I'm wrong or if there was a decision early in the development to provide every component with an ID?

Reply:
Just a speculation, but I'd think that the id's that appear in the debug statements are names of the touchables that are generated automatically. Readout id's are probably only for the sensitive volumes.